Denik CZ.js

```diff
@@ -0,0 +1,131 @@
+function detectWeb(doc, url) {
+	return "newspaperArticle";
+}
+
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+
+function scrape(doc, url) {
+	var newArticle = new Zotero.Item('newspaperArticle');
+	newArticle.url = url;
+	newArticle.title = ZU.trimInternal(ZU.xpathText(doc, '//div[@class = "dv3-clanek-left-fix"]/h1')).replace(/^,/, "");
```

Comment:
Move `ZU.trimInternal(ZU.xpathText(doc, '//div[@class = "dv3-clanek-left-fix"]/h1')).replace(/^,/, "")` to a function (e.g. `getTitle(doc)`) and call it from here. Then add a check in `detectWeb` that looks something like
```if (getTitle(doc)) return 'newspaperArticle';`